index.html

```diff
@@ -114,159 +92,132 @@ <h2>Custom software built for your everyday business needs.</h2>
+            <div class="icon"><i class="bi-file-earmark-binary"></i></div>
             <h4><a href="">System Automation</a></h4>
             <p>Authorative use of software, hardware and other tools to automate the administration, management, monitoring, and other operational tasks of computing. We have a strong understanding of a variety of programming languages and experience with automation tools and frameworks.</p>
           </div>
         </div>
-
         <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4 mt-lg-0" data-aos="fade-up" data-aos-delay="300">
           <div class="icon-box">
-            <div class="icon"><i class="bx bx-tachometer"></i></div>
+            <div class="icon"><i class="bi-cloud-download"></i></div>
             <h4><a href="">Legacy Software Intregration Development</a></h4>
             <p> We have good practices integrating legacy software systems with newer systems and technologies. This can involve integrating legacy systems with modern databases, cloud platforms, or other applications in order to make them more compatible and usable.</p>
           </div>
         </div>
-
         <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4" data-aos="fade-up" data-aos-delay="450">
           <div class="icon-box">
-            <div class="icon"><i class="bx bx-world"></i></div>
+            <div class="icon"><i class="bi-window-plus"></i></div>
             <h4><a href="">Web design</a></h4>
             <p>We have a strong understanding of both the technical and aesthetic aspects of web design, and are able to effectively balance these two elements to create web elements that are both functional and beautiful.</p>
           </div>
         </div>
-
         <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4" data-aos="fade-up" data-aos-delay="600">
           <div class="icon-box">
             <div class="icon"><i class="bx bx-slideshow"></i></div>
             <h4><a href="">System, BI Analytics</a></h4>
             <p>Here at NEAM, we take two related, but distint fields, and take statistical algorithms, and other techniques to extract insights and knowledge from data in order to support statistically smart decision-making.</p>
           </div>
-        </div>
-        
+        </div>  
         <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4" data-aos="fade-up" data-aos-delay="750">
           <div class="icon-box">
-            <div class="icon"><i class="bx bx-arch"></i></div>
+            <div class="icon"><i class="bi-filetype-ai"></i></div>
             <h4><a href="">A.I. Modeling</a></h4>
-              <p>(coming soon)</p>
+              <p>(Coming Soon)</p>
               <p>We plan on devloping advanced A.I models to help business crunch customer analytics and internal System analytics. Using A.I modeling, we would help business make better decisions in terms of predicting and showcasing the outcome and impact of those choice.</p>

```

Comment:
```suggestion
              <p>We plan on developing advanced A.I models to help business crunch customer analytics and internal System analytics. Using A.I modeling, we would help business make better decisions in terms of predicting and showcasing the outcome and impact of those choice.</p>

```
Developing isn't spelled right.